src/main/java/com/diamondfire/helpbot/sys/multiselector/MultiSelector.java

```diff
@@ -42,8 +41,11 @@ public void send(JDA jda) {
             buttons.add(button);
         }
         
-        jda.getTextChannelById(channel).sendMessageEmbeds(pages[0].getPage().build()).setActionRows(Util.of(buttons)).queue((message) -> {
-            ButtonHandler.addListener(user, message, event -> {
+        if (event instanceof SlashCommandEvent slashCommandEvent) {
+            slashCommandEvent.getInternalEvent().reply("The output of your command will be displayed below.").setEphemeral(true).queue();
+        }
+        event.getChannel().sendMessageEmbeds(pages[0].getPage().build()).setActionRows(Util.of(buttons)).queue((message) -> {
+            ButtonHandler.addListener(event.getMember().getIdLong(), message, event -> {
```

Comment:
Ah I see why.. yea we can't really do this here as you don't want to hold onto discord entities like this for a long time. Maybe we can make some kind of callback for this? At least I think there should be a callback somewhere? If not maybe we can make a multi selector for emphemral responses? donno